presto-main/src/test/java/com/facebook/presto/spiller/TestGenericPartitioningSpiller.java

```diff
@@ -139,6 +132,17 @@ private void assertSpilledPages(
         }
     }
 
+    private static AggregatedMemoryContext mockMemoryContext()
+    {
+        return new AggregatedMemoryContext();
+    }
+
+    private static SpillContext mockSpillContext()
+    {
+        return bytes -> {
```

Comment:
what does `{}` mean here? I know that this method is extracted from code which was here already, but still I do not understand this lambda.